index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <body>
+    <meta charset="UTF-8">
+    <title>This is th product landing page</title>
+    <link rel="stylesheet" href="css/style.css">
+    </head>
+    <header id="header">
+      <img id="header-img" src="https://www.google.com/search?q=arsenal+logo&sxsrf=AOaemvJ2iGf2sIpbR3WOhxB93YqR5I1RBw:1635447815377&tbm=isch&source=iu&ictx=1&fir=IY7y-tr6IfwpcM%252CxhkP-H2nIViUHM%252C_%253Bd-UumMlHiVIYaM%252CS6pmbXwDFvZGMM%252C_%253B4l38_zrewpzn4M%252CAfTiykD9PVqfmM%252C_%253B7PMGf2_BW0LfDM%252CS6pmbXwDFvZGMM%252C_%253BnHXfXXYBe_VP3M%252CTwg54SIo72q6qM%252C_%253BQYnyThlj75pEPM%252CD91IPADDblwofM%252C_%253BpP7HWTT9B_tRoM%252C8ir9UdMFgAtddM%252C_%253Bm2XXeuqwX7P4HM%252ChxeFZBXv_5TpCM%252C_%253B3RLmY8hZf3qNUM%252C6_n66LqdAWWr6M%252C_%253B9hGjK7NpOD_fbM%252Cv8-_NHQDLGtZkM%252C_%253BjnzopaeJE_PToM%252CNPofrOBoNOT-2M%252C_%253BUrDsjSrIyc9cOM%252CS6pmbXwDFvZGMM%252C_%253Bp5b2n00TZ0p6UM%252Cxq_5aXidk87VlM%252C_%253BoQL8SItDaO-_5M%252ClhS-9I51WNv_TM%252C_%253BFnpwrTvNGv5G5M%252C-BCdciEo0KSV7M%252C_&vet=1&usg=AI4_-kQccb-XYMel344wC0NCrhOGdI3BgA&sa=X&ved=2ahUKEwjy1NXw5e3zAhWniv0HHd4yAsQQ9QF6BAgREAE#imgrc=d-UumMlHiVIYaM">
```

Comment:
Please try to use the alt text for every images you are adding. Since this improves accessibility.